src/main/resources/models/tables/PlayerStatsView.hjson

```diff
@@ -0,0 +1,56 @@
+{
+    tables: [{
+        name: PlayerStatsView
```

Comment:
Lets change the name to PlayerStats here and in the file name to match the table name itself.